src/@types.d.ts

```diff
@@ -168,11 +168,20 @@ export type Tag = {
   isSpecial?: boolean;
 };
 
+export type OrderOptions = "title" | "artist" | "creator" | "bpm" | "duration" | "dateAdded";
+
+export type OrderDirection = "asc" | "desc";
+
+export type OrderType = {
```

Comment:
Is there a reason for appending "Type" to this and not just calling it `Order`?

Reply:
I just have the habit of adding "Type" or "Props" at the end of names
Do you want me to remove it?

Reply:
yea I'd remove it, it's redundant